common/app/views/fragments/header.scala.html

```diff
@@ -20,9 +20,7 @@
 
             @defining(Edition(request).id.toLowerCase()) { editionId =>
                 <a href="@LinkTo {/}"
-                class="@{if(editionId == "uk") {"new-header__logo news-header__logo-best-website"} else {
```

Comment:
is this meant to be included?

Reply:
Yes, we do not vary the SVG sizing on DCR based on the logo, so no need to add a specific class.